src/components/flipper.tsx

```diff
@@ -0,0 +1,31 @@
+import styles from '@/styles/Flipper.module.scss'
+import Image from 'next/image'
+import { useState } from 'react';
+import { FlipperImages } from '@/data/ice-age-texas/flipper/flipperData'
+
+export default function Flipper() {
+  const [isOpened, setIsOpened] = useState(new Array(FlipperImages.length).fill(false));
+
+  const toggleIsOpened = (index:any) => {
+    const newIsOpened = [...isOpened];
+    newIsOpened[index] = !newIsOpened[index];
+    setIsOpened(newIsOpened);
+  };
+  return (
+    <div className={styles.flipperWrapper}>
+      {FlipperImages.map((image, index) => (
+        <div key={index} className={styles.imageWrapper} onClick={() => toggleIsOpened(index)}>
+          <Image
+            src={isOpened[index] ? image.opened : image.closed}
+            className={styles.flipper}
+            alt={`Flipper ${index + 1}`}
+            layout="responsive"
```

Comment:
this has been deprecated and replaced with the `fill` property. (You should see an indicator in VSCode of this.)

Reply:
just removed it, doesn't seem necessary